qtwebkit/Source/WebKit2/UIProcess/API/qt/qquickwebview_p.h

```diff
@@ -344,6 +345,10 @@ class QWEBKIT_EXPORT QQuickWebViewExperimental : public QObject {
     QUrl userStyleSheet() const;
     void setUserStyleSheet(const QUrl& userStyleSheet);
     QUrl remoteInspectorUrl() const;
+    bool repositionInputField() const;
+    void setRepositionInputField(bool repositionInputField);
+
+    Q_INVOKABLE void focusEditableArea();
```

Comment:
Maybe something like... ```animateInputFieldVisible``` as it is already focused but not animated to the target area.